pynest/examples/Potjans_2014/network_params.py

```diff
@@ -168,91 +83,90 @@ def get_std_PSP_matrix(PSP_rel, number_of_pop):
              [0.1004, 0.0622, 0.0505, 0.0057, 0.0831, 0.3726, 0.0204, 0.],
              [0.0548, 0.0269, 0.0257, 0.0022, 0.06, 0.3158, 0.0086, 0.],
              [0.0156, 0.0066, 0.0211, 0.0166, 0.0572, 0.0197, 0.0396, 0.2252],
-             [0.0364, 0.001, 0.0034, 0.0005, 0.0277, 0.008, 0.0658, 0.1443]]
-    ),
-    # Number of external connections to the different populations.
-    # The order corresponds to the order in 'populations'.
+             [0.0364, 0.001, 0.0034, 0.0005, 0.0277, 0.008, 0.0658, 0.1443]]),
+    # number of external connections to the different populations (same order
+    # as in 'populations')
     'K_ext': np.array([1600, 1500, 2100, 1900, 2000, 1900, 2900, 2100]),
-    # Factor to scale the indegrees.
-    'K_scaling': 0.1,
-    # Factor to scale the number of neurons.
-    'N_scaling': 0.1,
-    # Mean amplitude of excitatory postsynaptic potential (in mV).
+    # mean amplitude of excitatory postsynaptic potential (in mV)
     'PSP_e': 0.15,
-    # Relative standard deviation of the postsynaptic potential.
-    'PSP_sd': 0.1,
-    # Relative inhibitory synaptic strength (in relative units).
+    # relative standard deviation of the postsynaptic potential
+    'PSP_std': 0.1,
+    # relative inhibitory synaptic strength
     'g': -4,
-    # Rate of the Poissonian spike generator (in Hz).
+    # rate of the Poissonian spike generator (in Hz)
     'bg_rate': 8.,
-    # Turn Poisson input on or off (True or False).
+    # turn Poisson input on or off (True or False)
+    # if False: DC input is applied for compensation
     'poisson_input': True,
-    # Delay of the Poisson generator (in ms).
+    # delay of the Poisson generator (in ms)
     'poisson_delay': 1.5,
-    # Mean delay of excitatory connections (in ms).
+    # mean delay of excitatory connections (in ms)
     'mean_delay_exc': 1.5,
-    # Mean delay of inhibitory connections (in ms).
+    # mean delay of inhibitory connections (in ms)
```

Comment:
I'm okay with the naming mean_delay_exc, mean_delay_inh, V0_mean, but if you want to consistently have 'mean' either in front or at the back, I would choose at the back: delay_exc_mean etc.

Reply:
I renamed and used the back as you suggested.